Framework/PythonInterface/mantid/plots/__init__.py

```diff
@@ -535,7 +545,16 @@ def errorbar(self, *args, **kwargs):
         if helperfunctions.validate_args(*args):
             logger.debug('using plotfunctions')
 
+            autoscale_on_update = kwargs.pop("autoscale_on_update", True)
+
             def _data_update(artists, workspace, new_kwargs=None):
+                if self.lines:
+                    # If we have at least one line, set autoscaling
+                    # based on autoscale_on_update flag.
+                    # If we have no lines currently, we want to autoscale
+                    # regardless
```

Comment:
Just a small bit of clarification for how this behaves when only one line is plotted.

```suggestion
                    # regardless. 
                    # If there is only one line and the data is changed
                    # the plot will autoscale or not based on the flag 
                    # of the one plotted line.
```

Reply:
Good suggestion. I've moved the comment to `plot`, rather than `errorbar` as I think it would be a more visible place to put it

Reply:
Yep that seems all good to me.